index.html

```diff
@@ -10,55 +10,50 @@
 </head>
 
 <body>
-    <!-- home page header -->
+    <header>
+        
+        <!-- Navigation  bar containr-->
+        <nav class="navbar">
+            <a class="nav-logo" href="">My Logo</a>
+            <!-- static menu button -->
+            <ul class="nav-menu">
+                <li class="nav-item"><a class="nav-link" href="#portfolio-section" onclick="closeItem()">Portfolio</a></li>
+                <li class="nav-item"><a class="nav-link" href="#about-section" onclick="closeItem()">About</a></li>
+                <li class="nav-item"><a class="nav-link" href="#contact-section" onclick="closeItem()">Contact</a></li>
```

Comment:
- [ ] i don't see any use or source of the `closeItem()` function , for best javascript and HTML practices kindly remove it from the nav links since it has no functionalities to keep your your code clean and professional 💯